website/docs/components/stepper/partials/code/showcase.md

```diff
@@ -1,3 +1,5 @@
+## Showcase
+
```

Comment:
Btw I've found that if you change `@model` to `this.` in the code for iterating through examples it fixes the display.

Reply:
Good call. I pushed up a commit with that.